R/plotMetExp.R

```diff
@@ -0,0 +1,74 @@
+#' plotMetExp
+#' This function visualizes results of EpiMix. 
+#' @param EpiMix_results The object, a list, that was returned from
+#' running the EpiMix function and is one of the outputs from the 
+#' GMA function.
+#' @param probe_name A character string containing the name of the
+#' CpG probe that will be plotted. 
+#' @param dataMET A data matrix containing the methylation data
+#' where the CpG probes are in the rows and samples are in the
+#' columns
+#' @param dataEXP A data matrix containing the gene expression
+#' data where the genes are in the rows and the samples are in 
+#' the columns
+#' @param gene_of_interest A character string containing the 
+#' name of the gene that will be plotted. 
+#' @param additionalFilename A character string that can be used
+#' to add a prefix or filepath to the filename of the pdf 
+#' visualizing the heatmap. Default is an empty string. 
+#' @importFrom EpiMix EpiMix_PlotModel
+#' @importFrom ggplot2 ggsave
+#' @return No value is returned. Visualizations are saved. 
+#' @export
+#' @examples 
+#' data("EpiMix_Results_Regular")
+#' data("dataMethyl")
+#' data("dataFiltCol")
+#' plotMetExp(EpiMix_results = EpiMix_Results_Regular,
+#'            probe_name = "cg03035704",
+#'            dataMET = dataMethyl,
+#'            dataEXP = dataFiltCol,
+#'            gene_of_interest = "ACVRL1",
+#'            additionalFilename = "./GMAresults/")
+plotMetExp <- function(EpiMix_results,
+                       probe_name,
+                       dataMET,
+                       dataEXP,
+                       gene_of_interest,
+                       additionalFilename = "") {
```

Comment:
same as before re `additionalFilename`

Reply:
I've kept it as additionalFilename for consistency with other Moonlight functions but let me know if I should change it here